src/components/Menu/Menu/__tests__/Menu.interactions.js

```diff
@@ -0,0 +1,81 @@
+import { userEvent } from "@storybook/testing-library";
+import {
```

Comment:
@hadasfa  from what I see it *is* the name convention 🤔 
![image](https://user-images.githubusercontent.com/96776835/160758088-57825b84-300c-4d96-986d-2a9cd6cc6ec8.png)

I think it's ok since the test files also have a dot between chunks in their name
![image](https://user-images.githubusercontent.com/96776835/160758191-f171bc1d-4fbe-4048-ab45-4beb37b89479.png)

Reply:
updated as discussed :)